login/index.html

```diff
@@ -10,14 +10,18 @@
   </script>
 </head>
 <body>
-  <div>
-      <label for="username">Username: </label>
-      <input type="text" id="username"></div>
-  <div>
-      <label for="password">Password: </label>
-      <input type="password" id="password">
+<div id="login-window-title"><span>Log In to SimpleApp</span></div>
```

Comment:
Things that appear only once may have an id, things that appear more than once only a class